hubstorage/serialization.py

```diff
@@ -1,4 +1,5 @@
 from json import dumps, loads
+from msgpack import Unpacker
```

Comment:
IMO we should enforce `msgpack` when possible (ie, when the endpoint supports it). Some benchmarks that I've done has shown that we can have significant performance gains, specially with `/items` and `/collections` that may be huge in some cases.